### PR TITLE
Add quizRouter and userRouter and json

### PR DESCRIPTION
**Pull request by GreenGard**

### CODE REVIEW

File: backend/routes/index.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import { defaultRoute } from './defaultRoute';
+import {quizRoute} from './quiz';
+import {userRoute} from './user';
+
+export const routes = express.Router();
+
+routes.use(defaultRoute);
+routes.use(quizRoute);
+routes.use(userRoute);
```
Comment (DarkendHall) [inline backend/routes/index.ts:10]:
```suggestion
routes.use(userRoute);

```

File: backend/routes/defaultRoute.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { Router } from 'express';
+
+export const defaultRoute = Router();
+
+defaultRoute.get('/', (_req, res) => {
+    res.send('What\'s up doc ?!');
+});
```
Comment (DarkendHall) [inline backend/routes/defaultRoute.ts:7]:
```suggestion
});

```

File: backend/json/questions.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": 1,
+  "question": "Var ligger paris?",
+  "answers": "hej,nej.hejsan",
+  "correctAnswer": "hej",
+  "continent": "EContinent"
+
+}
```
Comment (DarkendHall) [inline backend/json/questions.json:8]:
```suggestion
}

```

File: backend/app.ts [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
+// routes
+app.use('/', routes);
 
-export default {};
+// start the server
+app.listen(3000, () => {
+    console.log(
+        `server running : 'http://localhost:3000`
+    );
+});
```
Comment (DarkendHall) [inline backend/app.ts:15]:
```suggestion
});

```

File: frontend/src/router/index.ts [hunk 5]
```diff
@@ -9,6 +9,7 @@ const router = createRouter({
             name: 'home',
             component: HomeView,
         },
+
```
Comment (DarkendHall) [inline frontend/src/router/index.ts:12]: ?

```suggestion
```

**Review by DarkendHall — CHANGES_REQUESTED** [review]

**Review by OscarStjernfeldt — APPROVED** [review]

**Pull request merged**